diff --git a/Mopy/bash/bash.py b/Mopy/bash/bash.py
index 0dd54eca6..0aca111b3 100644
--- a/Mopy/bash/bash.py
+++ b/Mopy/bash/bash.py
@@ -82,7 +82,11 @@ def _print(s):
     if pathToProg:
         os.chdir(pathToProg)
     global _bugdump_handle
-    _bugdump_handle = open(os.path.join(os.getcwd(), 'BashBugDump.log'), 'w',
+    bugdump_base = (os.getcwd() if bolt.os_name == 'nt' else
+                    os.path.join(os.path.expanduser('~'), '.var', 'app',
+                        'io.github.wrye_bash.wrye_bash', 'data', 'wrye_bash'))
+    os.makedirs(bugdump_base, exist_ok=True)
+    _bugdump_handle = open(os.path.join(bugdump_base, 'BashBugDump.log'), 'w',
         buffering=1, encoding='utf-8')
     _install_bugdump()
 
