diff --git a/Mopy/bash/barb.py b/Mopy/bash/barb.py
index 6905768c6..8daed1129 100644
--- a/Mopy/bash/barb.py
+++ b/Mopy/bash/barb.py
@@ -61,7 +61,6 @@ def _init_settings_files(bak_name, mg_name, root_prefix, mods_folder_name):
     if not initialization.bash_dirs_initialized:
         raise BoltError(u'_init_settings_files: Bash dirs are not initialized')
     settings_info = {
-        (dirs[u'mopy'], jo(bak_name, u'Mopy')): {u'bash.ini', },
         (dirs['mods'].join('Bash'), jo(bak_name, mods_folder_name, 'Bash')): {
             'Table.dat', },
         (dirs['mods'].join('Docs'), jo(bak_name, mods_folder_name, 'Docs')): {
@@ -78,9 +77,8 @@ def _init_settings_files(bak_name, mg_name, root_prefix, mods_folder_name):
            u'Converters.dat', u'Installers.dat', },
         (dirs[u'saveBase'], jo(u'My Games', mg_name)): {
             u'BashProfiles.dat', u'BashSettings.dat', u'BashLoadOrders.dat'},
-        # backup all files in Mopy\bash\l10n, Data\Bash Patches\,
+        # backup all files in Data\Bash Patches\,
         # Data\BashTags\ and Data\INI Tweaks\
-        (dirs[u'l10n'], jo(bak_name, u'Mopy', u'bash', u'l10n')): {},
         (dirs['mods'].join('Bash Patches'),
          jo(bak_name, mods_folder_name, 'Bash Patches')): {},
         (dirs['mods'].join('BashTags'),
